tools/twix/src/panels/plot.rs

```diff
@@ -91,14 +101,75 @@ impl LineData {
                                 let value = lua_function
                                     .call::<_, f64>(self.lua.to_value(value))
                                     .unwrap_or(f64::NAN);
-                                [i as f64, value]
+                                [(buffer_size + i - buffered_values.len()) as f64, value]
```

Comment:
I'm not sure if it works. But doesn't this get much simpler by dropping the `.rev()` and starting to add plot points from the back? Is it okay to have that for `PlotPoints`?

Reply:
Yes, this makes things a bit easier to read, thanks.